src/nvim/CMakeLists.txt

```diff
@@ -65,6 +65,10 @@ foreach(sfile ${NEOVIM_SOURCES})
   if(${f} MATCHES "^(regexp_nfa.c)$")
```

Comment:
I would name such files “foo.c.h”. And @equalsraf is right, this is a “part” of regexp.c and cannot be compiled on its own.